packages/aws-rfdk/package.json

```diff
@@ -65,145 +65,146 @@
-    "@aws-cdk/aws-cloudfront": "1.108.1",
-    "@aws-cdk/aws-cloudwatch": "1.108.1",
-    "@aws-cdk/aws-cloudwatch-actions": "1.108.1",
-    "@aws-cdk/aws-codebuild": "1.108.1",
-    "@aws-cdk/aws-codecommit": "1.108.1",
-    "@aws-cdk/aws-codeguruprofiler": "1.108.1",
-    "@aws-cdk/aws-codepipeline": "1.108.1",
-    "@aws-cdk/aws-cognito": "1.108.1",
-    "@aws-cdk/aws-databrew": "1.108.1",
-    "@aws-cdk/aws-docdb": "1.108.1",
-    "@aws-cdk/aws-dynamodb": "1.108.1",
-    "@aws-cdk/aws-ec2": "1.108.1",
-    "@aws-cdk/aws-ecr": "1.108.1",
-    "@aws-cdk/aws-ecr-assets": "1.108.1",
-    "@aws-cdk/aws-ecs": "1.108.1",
-    "@aws-cdk/aws-ecs-patterns": "1.108.1",
-    "@aws-cdk/aws-efs": "1.108.1",
-    "@aws-cdk/aws-eks": "1.108.1",
-    "@aws-cdk/aws-elasticloadbalancing": "1.108.1",
-    "@aws-cdk/aws-elasticloadbalancingv2": "1.108.1",
-    "@aws-cdk/aws-events": "1.108.1",
-    "@aws-cdk/aws-events-targets": "1.108.1",
-    "@aws-cdk/aws-globalaccelerator": "1.108.1",
-    "@aws-cdk/aws-glue": "1.108.1",
-    "@aws-cdk/aws-iam": "1.108.1",
-    "@aws-cdk/aws-kinesis": "1.108.1",
-    "@aws-cdk/aws-kinesisfirehose": "1.108.1",
-    "@aws-cdk/aws-kms": "1.108.1",
-    "@aws-cdk/aws-lambda": "1.108.1",
-    "@aws-cdk/aws-logs": "1.108.1",
-    "@aws-cdk/aws-route53": "1.108.1",
-    "@aws-cdk/aws-route53-targets": "1.108.1",
-    "@aws-cdk/aws-s3": "1.108.1",
-    "@aws-cdk/aws-s3-assets": "1.108.1",
-    "@aws-cdk/aws-sam": "1.108.1",
-    "@aws-cdk/aws-secretsmanager": "1.108.1",
-    "@aws-cdk/aws-servicediscovery": "1.108.1",
-    "@aws-cdk/aws-signer": "1.108.1",
-    "@aws-cdk/aws-sns": "1.108.1",
-    "@aws-cdk/aws-sns-subscriptions": "1.108.1",
-    "@aws-cdk/aws-sqs": "1.108.1",
-    "@aws-cdk/aws-ssm": "1.108.1",
-    "@aws-cdk/aws-stepfunctions": "1.108.1",
-    "@aws-cdk/aws-stepfunctions-tasks": "1.108.1",
-    "@aws-cdk/cloud-assembly-schema": "1.108.1",
-    "@aws-cdk/core": "1.108.1",
-    "@aws-cdk/custom-resources": "1.108.1",
-    "@aws-cdk/cx-api": "1.108.1",
-    "@aws-cdk/lambda-layer-awscli": "1.108.1",
-    "@aws-cdk/lambda-layer-kubectl": "1.108.1",
-    "@aws-cdk/region-info": "1.108.1",
+    "@aws-cdk/assets": "1.111.0",
+    "@aws-cdk/aws-apigateway": "1.111.0",
+    "@aws-cdk/aws-apigatewayv2": "1.111.0",
+    "@aws-cdk/aws-apigatewayv2-integrations": "1.111.0",
+    "@aws-cdk/aws-applicationautoscaling": "1.111.0",
+    "@aws-cdk/aws-autoscaling": "1.111.0",
+    "@aws-cdk/aws-autoscaling-common": "1.111.0",
+    "@aws-cdk/aws-autoscaling-hooktargets": "1.111.0",
+    "@aws-cdk/aws-batch": "1.111.0",
+    "@aws-cdk/aws-certificatemanager": "1.111.0",
+    "@aws-cdk/aws-cloudformation": "1.111.0",
+    "@aws-cdk/aws-cloudfront": "1.111.0",
+    "@aws-cdk/aws-cloudwatch": "1.111.0",
+    "@aws-cdk/aws-cloudwatch-actions": "1.111.0",
+    "@aws-cdk/aws-codebuild": "1.111.0",
+    "@aws-cdk/aws-codecommit": "1.111.0",
+    "@aws-cdk/aws-codeguruprofiler": "1.111.0",
+    "@aws-cdk/aws-codepipeline": "1.111.0",
```

Comment:
Missed adding `@aws-cdk/aws-codestarnotifications": "1.111.0",` here. deps & peer deps are in-sync.

Reply:
Fixed